docs/git/from-desktop-to-cli-cheatsheet.md

```diff
@@ -0,0 +1,85 @@
+---
+id: cheatsheet-cli
+title: From Github Desktop To CLI
+sidebar_label: Desktop to CLI Cheatsheet
+---
+
+Everything that you're going to be doing using Git CLI you can also do using Github Desktop. Let's run through a few example of this
+
+### Viewing Changed Files
+
+To view the files that you have changed you can use
+
+```sh
+git status
+```
+
+This will show a list of files that have changed since the last time that you committed.
+
+| Github Desktop                                                 | Git CLI                                                      |
+| -------------------------------------------------------------- | ------------------------------------------------------------ |
+| ![Viewing changed files in Github Desktop](assets/changed.png) | ![Viewing changed files in Git CLI](assets/changing-cli.png) |
+
+### Including Files to Commit
+
+One key different between Github Desktop and Git CLI is that you need to set which files will be committed to your repository. You can add files to be committed by using
+
+```sh
+git add .
+```
+
+This command will add **every** file you have changed to be ready to commit
+
+| Github Desktop                                      | Git CLI                                                          |
+| --------------------------------------------------- | ---------------------------------------------------------------- |
+| !["Adding files to commit"](assets/addingfiles.gif) | !["Adding files to commit with cli"](assets/addingfiles-cli.gif) |
+
+### Committing Changes
+
+Committing changes to your repository can be done by using this command
+
+```sh
+git commit -m "YOUR MESSAGE HERE"
+```
+
+The `-m` means that after you are going to write your commit message.
```

Comment:
```suggestion
The `-m` means that after you are going to write your commit message. Make sure to put your commit message in quotes! You can press enter to add new lines in your message, too.

Some characters are treated specially in the terminal, and we can tell the terminal not to treat them specially by _escaping_ them with a backslash. For example, if you want to use a quote character inside your message, you need to escape it, otherwise the terminal will think your quote is the end of the message:

```sh
git commit -m "Use \"s instead of 's" 
```

One of the other characters that needs escaping is the exclamation mark - if you want to use an exclamation mark in a commit message, make sure to escape it:

```sh
git commit -m "Make it prettier\!"
```
```

Reply:
Ugh github isn't great at suggestions that include code blocks :D If you try to edit this message, you should see what I was trying to suggest...